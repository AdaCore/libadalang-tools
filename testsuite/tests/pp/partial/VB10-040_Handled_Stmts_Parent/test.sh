#! /usr/bin/env bash

# Single statement
partial_gnatpp -P default.gpr -S main.adb -SL 6 -SC 6 -EL 7 -EC 7
# Multiple statements with pre and post comments
partial_gnatpp -P default.gpr -S main.adb -SL 5 -SC 6 -EL 11 -EC 7



