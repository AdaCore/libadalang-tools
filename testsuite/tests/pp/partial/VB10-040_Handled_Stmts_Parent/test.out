*************************************
main.adb(<AssignStmt main.adb:6:4-7:9>) - 6:1-7:9
^
   A := 1;$
*************************************

*************************************
main.adb(<StmtList main.adb:6:4-10:9>) - 5:1-11:15
^
   -- Comment
   A := 1;
   -- Comment;
   B := 2;
   -- Comment;$
*************************************

